### PR TITLE
Close changelog for 7.8.1

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

Update Changelogs

## Checklist

- ~~[ ] My code follows the style guidelines of this project~~
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

File: libbeat/docs/release.asciidoc [hunk 1]
```diff
@@ -34,6 +36,8 @@ upgrade.
 * <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
+* <<release-notes-6.8.11>>
+* <<release-notes-6.8.10>>
```
Comment (urso) [inline libbeat/docs/release.asciidoc:40]: Are these sections already in the Changelog.asciidoc? You might need to copy 6.8.10 and 6.8.11 changelog to the 7.8, 7.x, and master branch.

Comment (kuisathaverat) [reply libbeat/docs/release.asciidoc:40]: I suspected something was missing

Comment (kuisathaverat) [reply libbeat/docs/release.asciidoc:40]: Every branch has a different changelog, even the master branch does not have all release notes versions, I will rid of that part

Comment (kuisathaverat) [inline libbeat/docs/release.asciidoc:40]:
```suggestion
```

File: libbeat/docs/release.asciidoc [hunk 2]
```diff
@@ -8,7 +8,9 @@ This section summarizes the changes in each release. Also read
 <<breaking-changes>> for more detail about changes that affect
 upgrade.
 
+* <<release-notes-7.8.1>>
 * <<release-notes-7.8.0>>
+* <<release-notes-7.7.1>>
```
Comment (kuisathaverat) [inline libbeat/docs/release.asciidoc:13]:
```suggestion
```

File: libbeat/docs/release.asciidoc (hunk 1 above)
Comment (urso) [reply libbeat/docs/release.asciidoc]: > Every branch has a different changelog

The changelog in each branch must be held consistent. The changelog and documentation is generated from the release branch. This means that 7.8 release branch must include all changelogs for past 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, ... 6.8, ... releases

The 7.8 branch will contain all changelogs up to the most recent 7.8.x patch release.
The 7.9 branch will contain all changelogs up to the most recent 7.9.x patch release.

The changelogs in the 7.x and master branch need to be kept in sync with the actual release branches.

If the changelogs in the branches differ, then we're missing the required updates in some branches.

**Comment by kuisathaverat** [comment]
@urso can you take a last look to this.

**Comment by urso** [comment]
The 7.8.1 changes look good. We still need to add 7.7.1 and old 6.8.10 changelog. The release notes are published here per release branch: https://www.elastic.co/guide/en/beats/libbeat/current/release-notes.html

comparing 7.8 and 7.7 or 6.8 branch the notes are currently inconsistent:
- 7.8: https://www.elastic.co/guide/en/beats/libbeat/7.8/release-notes.html
- 7.7: https://www.elastic.co/guide/en/beats/libbeat/7.7/release-notes.html
- 6.8: https://www.elastic.co/guide/en/beats/libbeat/6.8/release-notes.html

The 7.8 branch is missing at least 7.7.1 (which was release before 7.8.0) and there have been added some minor release we're still missing in the changelog. The 7.x and master changelogs also need some cleanup. Do we plan to cleanup the 7.8 changelog in a follow up PR?

**Comment by kuisathaverat** [comment]
@urso We noticed the differences between release notes and the gaps in some of them, we can open an issue for that and fix all the changelogs in a follow-up. So if the changelog in this PR is correct, we could merge it and fix the changelog issues later

**Comment by urso** [comment]
> @urso We noticed the differences between release notes and the gaps in some of them, we can open an issue for that and fix all the changelogs in a follow-up. So if the changelog in this PR is correct, we could merge it and fix the changelog issues later

SGTM. Let's do this.

**Pull request merged**